index.html

```diff
@@ -29,6 +29,17 @@
                     </g>
                 </svg>
             </a>
+            <div class="nav-links">
+                <ul class="nav-links">
+                    <li><a href="#">Portfolio</a></li>
+                    <li><a href="#">About</a></li>
+                    <li><a href="#">Contact</a></li>
+                    <li><a href="#"><svg width="22" height="18" viewBox="0 0 22 18" xmlns="http://www.w3.org/2000/svg">
+                                <path fill-rule="evenodd" clip-rule="evenodd"
+                                    d="M6.4 2H15.6C16.7531 2 17.4977 2.00156 18.0643 2.04785C18.6077 2.09225 18.8091 2.1676 18.908 2.21799C19.2843 2.40973 19.5903 2.71569 19.782 3.09202C19.8324 3.19091 19.9078 3.39235 19.9521 3.93567C19.9984 4.50235 20 5.24689 20 6.4V11.6C20 12.7531 19.9984 13.4977 19.9521 14.0643C19.9078 14.6077 19.8324 14.8091 19.782 14.908C19.5903 15.2843 19.2843 15.5903 18.908 15.782C18.8091 15.8324 18.6077 15.9078 18.0643 15.9521C17.4977 15.9984 16.7531 16 15.6 16H6.4C5.24689 16 4.50235 15.9984 3.93567 15.9521C3.39235 15.9078 3.19091 15.8324 3.09202 15.782C2.71569 15.5903 2.40973 15.2843 2.21799 14.908C2.1676 14.8091 2.09225 14.6077 2.04785 14.0643C2.00156 13.4977 2 12.7531 2 11.6V6.4C2 5.24689 2.00156 4.50235 2.04785 3.93567C2.09225 3.39235 2.1676 3.19091 2.21799 3.09202C2.40973 2.71569 2.71569 2.40973 3.09202 2.21799C3.19091 2.1676 3.39235 2.09225 3.93567 2.04785C4.50235 2.00156 5.24689 2 6.4 2ZM0 6.4C0 4.15979 0 3.03969 0.435974 2.18404C0.819467 1.43139 1.43139 0.819467 2.18404 0.435974C3.03969 0 4.15979 0 6.4 0H15.6C17.8402 0 18.9603 0 19.816 0.435974C20.5686 0.819467 21.1805 1.43139 21.564 2.18404C22 3.03969 22 4.15979 22 6.4V11.6C22 13.8402 22 14.9603 21.564 15.816C21.1805 16.5686 20.5686 17.1805 19.816 17.564C18.9603 18 17.8402 18 15.6 18H6.4C4.15979 18 3.03969 18 2.18404 17.564C1.43139 17.1805 0.819467 16.5686 0.435974 15.816C0 14.9603 0 13.8402 0 11.6V6.4ZM5.5547 4.16795C5.09517 3.8616 4.4743 3.98577 4.16795 4.4453C3.8616 4.90483 3.98577 5.5257 4.4453 5.83205L10.4453 9.83205C10.7812 10.056 11.2188 10.056 11.5547 9.83205L17.5547 5.83205C18.0142 5.5257 18.1384 4.90483 17.8321 4.4453C17.5257 3.98577 16.9048 3.8616 16.4453 4.16795L11 7.79815L5.5547 4.16795Z" />
+                            </svg></a></li>
+                </ul>
+            </div>
```

Comment:
Kindly align the nav elements and the rest of the section as the given design

### Original design 
![right1](https://user-images.githubusercontent.com/58376020/122475205-9b3df280-cfc4-11eb-8971-dff2bd5cb346.png)

#### Your project
![wrong1](https://user-images.githubusercontent.com/58376020/122475241-a8f37800-cfc4-11eb-8469-66dd8d14f98c.png)